### PR TITLE
ADD APM32 MCU USB_DFU

### PR DESCRIPTION
**Pull request by Alden-wrk**
By modifying port_handler.js and manifest.json, you can add the VendorID and ProductID corresponding to APM32 so that it can directly support the firmware download of APM32 series MCU，https://global.geehy.com/
Verified on APM32F405

### CODE REVIEW

**Comment by nerdCopter** [comment]
I don't know anything on the subject, are APM32 binary compatible with STM32?
we do not have AMP32 drivers whatsoever.

**Comment by Alden-wrk** [comment]
> I don't know anything on the subject, are APM32 binary compatible with STM32? we do not have AMP32 drivers whatsoever.

Yes, I have tested that APM32 is binary compatible with stm32, there are no problems in the flight test, using APM32 is more cost-effective.

**Review by nerdCopter — APPROVED** [review]

File: src/js/port_handler.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ const TIMEOUT_CHECK = 500; // With 250 it seems that it produces a memory leak a
 
 var usbDevices = { filters: [
     {'vendorId': 1155, 'productId': 57105},
-    {'vendorId': 10473, 'productId': 393}
+    {'vendorId': 10473, 'productId': 393},
+    {'vendorId': 12619, 'productId': 262}// APM32 DFU Bootloader
```
Comment (nerdCopter) [inline src/js/port_handler.js:8]:
```suggestion
    {'vendorId': 12619, 'productId': 262} // APM32 DFU Bootloader
```

**Comment by nerdCopter** [comment]
thank you! will merge after build success.

**Pull request merged**